Add step to enable repository for puppet-agent rpm (#1636)
theforeman/foreman-documentation: guides/common/modules/proc_installing-and-configuring-the-puppet-agent.adoc

.Prerequisites
* The host must have a Puppet environment assigned to it.
ifdef::satellite[]

> * The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host. The `puppet-agent rpm` is available on following repositories:

Reviewer: ```suggestion
* The {project-client-name} repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
The `puppet-agent` package is available in following repositories:
```

** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 - `{RepoRHEL7ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 8 Server RPMs x86_64 - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 9 Server RPMs x86_64 - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
+

> To enable required repository enter:

Reviewer: Please reword.

[options="nowrap", subs="+quotes,verbatim,attributes"]
+
----

> # subscription-manager repos --enable _repositoryName_

Reviewer: ```suggestion
# subscription-manager repos --enable _My_Repository_Name_
```

* The host must have a Puppet environment assigned to it.
ifdef::satellite[]
* The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.

> The `puppet-agent rpm` is available within the following repositories:

Reviewer 2: ```suggestion
The `puppet-agent` rpm is available within the following repositories:
```

** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 - `{RepoRHEL7ServerSatelliteToolsProductVersion}`

> ** {Team} {project-client-name} for RHEL 8 Server RPMs x86_64 - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
> ** {Team} {project-client-name} for RHEL 9 Server RPMs x86_64 - `{RepoRHEL9ServerSatelliteToolsProductVersion}`

Reviewer 2: Reading through the rendered result, this is inconsistent. Copying for simplicity:
```
Red Hat Satellite Client 6 for RHEL 8 Server RPMs x86_64 - satellite-client-6-for-rhel-8-<arch>-rpms
Red Hat Satellite Client 6 for RHEL 9 Server RPMs x86_64 - satellite-client-6-for-rhel-9-<arch>-rpms
```

I think this would be better.

```suggestion
** {Team} {project-client-name} for RHEL 8 Server RPMs <arch> - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 9 Server RPMs <arch> - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
```

* The host must have a Puppet environment assigned to it.
ifdef::satellite[]
* The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.

> The `puppet-agent rpm` is available on following repositories:

Reviewer: ```suggestion
The `puppet-agent rpm` is available within the following repositories:
```

** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 - `{RepoRHEL7ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 8 Server RPMs x86_64 - `{RepoRHEL8ServerSatelliteToolsProductVersion}`
** {Team} {project-client-name} for RHEL 9 Server RPMs x86_64 - `{RepoRHEL9ServerSatelliteToolsProductVersion}`
+
To enable the required repository, enter:
[options="nowrap", subs="+quotes,verbatim,attributes"]
+
----
# subscription-manager repos --enable _My_Repository_Name_

> ----

Reviewer 2: @ehelms can you have a look? I think the explicit enablement is redundant if the user has synced the client repo and enabled it in a content view for the host.

Reviewer 3: Given the prerequisites and how SCA tends to work these days I do think its redundant.

Author: I am not sure what changes you are suggesting. Could you please elaborate on the changes.

Reviewer 3: The suggestion is to drop lines 21-26
